docs/www/redpanda-nodejs.md

```diff
@@ -0,0 +1,177 @@
+##Node.js set up
```

Comment:
Change this to a h1.  #

Reply:
Also change the filename to:
`quick-start-nodejs`

Roko has named all the other files with this pattern:
https://github.com/vectorizedio/redpanda/tree/dev/docs/www

Reply:
got it